java-speech/google-cloud-speech/src/test/java/com/google/cloud/speech/v1/it/ITSpeechTest.java

```diff
@@ -44,17 +46,22 @@ public class ITSpeechTest {
 
   @BeforeClass
   public static void setupClass() throws Exception {
-    SpeechSettings.Builder settingsBuilder = SpeechSettings.newBuilder();
-    settingsBuilder
-        .longRunningRecognizeSettings()
-        .setRetrySettings(
+    TimedRetryAlgorithm timedRetryAlgorithm =
+        OperationTimedPollAlgorithm.create(
+            // These values are copied from com.google.cloud.speech.v1.stub.SpeechStubSettings...
             RetrySettings.newBuilder()
-                .setRetryDelayMultiplier(1.0)
-                .setMaxRpcTimeout(Duration.ofMinutes(10))
-                .setInitialRetryDelay(Duration.ofSeconds(5))
-                .setMaxAttempts(5)
-                .setMaxRetryDelay(Duration.ofSeconds(5))
+                .setInitialRetryDelay(Duration.ofMillis(500L))
+                .setRetryDelayMultiplier(1.5)
+                .setMaxRetryDelay(Duration.ofMillis(5000L))
+                .setInitialRpcTimeout(Duration.ZERO)
+                .setRpcTimeoutMultiplier(1.0)
+                .setMaxRpcTimeout(Duration.ZERO)
+                .setTotalTimeout(Duration.ofDays(1))
```

Comment:
This seems like a pretty long timeout.

Reply:
I believe the default timeout set to be 5 min. I think you can probably just double it to 10 minutes and *should* be fine. There was an issue from the java-doc-samples repo a few weeks ago about a similar LRO issue and the runtime recently increased for ~4:30min to ~5:30min.

Reply:
Using a more reasonable `ofHours(1)`

Reply:
I did not see your suggestion @lqiu96 . Let's try 10 minutes